README.md

```diff
@@ -67,9 +67,21 @@ to revise it in response.
 * Build consensus and integrate feedback. RFCs that have broad support
 are much more likely to make progress than those that don't receive any
 comments.
-* Eventually, somebody on the [core team] will either accept the RFC by
-merging the pull request and assigning the RFC a number, at which point
-the RFC is 'active', or reject it by closing the pull request.
+* Eventually, the [core team] will decide whether the RFC is a candidate
+for inclusion in Ember.
+* RFCs that are candidates for inclusion in Ember will enter a "final comment
+period", which shall be no shorter than 7 days. This period will be signaled
+with a comment and tag on the RFC's pull request. Furthermore,
+[Ember's official Twitter account](https://twitter.com/emberjs) will post a
+tweet about the RFC to attract the community's attention.
+* An RFC can be modified based upon feedback from the [core team] and community.
+Significant modifications may trigger a new final comment period.
+* An RFC can be rejected at any point during this process. A rejected RFC will
+have its associated pull request closed and a member of the [core team] will
+explain the rationale for rejection.
```

Comment:
An RFC should only be rejected on the basis of rationale that has been fully debated (and reached a steady state) on the thread. We should make that explicit.